diagnostic.md

```diff
@@ -18,33 +18,32 @@ In what file does the above code go?
 In your own words, what does the command `bundle exec rake db:migrate` do?
 
 ```md
-// your response here
+db:migrate checks which missing migrations still need to be applied to the database without caring about the previouse ones.
 ```
 
 What is the difference between `rails g scaffold things` and
 `rails g controller things`?
 
 ```md
-// your response here
-```
+when you scaffold you create 3 files controllers model and serializer```
```

Comment:
`rails g scaffold things` will generate a migration, model, routes, serializer, controller, and tests for `things`.
`rails g controller things` will generate a controller and tests for `things`.